plot_app/templates/footer.html

```diff
@@ -1,23 +1,19 @@
 
     <footer id="footer">
-            <div class="row">
-                <div class="col-sm-12">
-                    <div class="copyright-text text-center">
-                        <p>&copy; 2019 PX4 Team. Source on <a
-                         href="https://github.com/PX4/flight_review">github</a>.
-                         Theme by <a target="_blank" href="https://bootswatch.com">Bootswatch</a>.
-{% if is_plot_page or is_stats_page %}
-                         Plotted with the awesome <a target="_blank"
-                         href="http://bokeh.pydata.org">bokeh</a> library.
-{% endif %}
-                        </p>
-                    </div>
-                </div>
-            </div>
+      <div class="row">
+        <div class="col-sm-12">
+          <div class="copyright-text text-center">
+            <p>Maps brought to you by <a target="_blank" href="https://www.mapbox.com/">MapBox</a>.</p>
+            <p>3D Tiles and Viewer provided by <a target="_blank" href="https://www.mapbox.com/">CesiumJS</a>.</p>
+            <p><br></p>
+            <p class="sm">UI Theme (Yeti) available from <a target="_blank" href="https://bootswatch.com/">Bootswatch</a></p>
```

Comment:
Can you add a `.` at the end here too, and below as well? Or remove them all, to be consistent.